A couple of changes

antim0118 opened the pull request:
RU: Мои изменения в папке Code, остальное делал Unity после обновления до 2018.3.12
- возможность указать путь к папке с дополнительными файлами (допустим, нужно распаковать vpk файлы и не засорять основную папку с игрой)
- исправлен редкий случай ошибки при парсинге vmt файлов (разрабы ксго криво форматируют свои файлы)
- изменение размера пропов в соответствии с UniformScale (при версии пропов - v11)

antim0118 committed:
Project updated to 2018.3.12f1

antim0118 committed:
Assets (additional) path

RU: возможность указать путь к папке с дополнительными файлами (допустим, нужно распаковать vpk файлы и не засорять основную папку с игрой)

antim0118 committed:
Fixed rare case of parsing vmt error

RU: исправлен редкий случай ошибки при парсинге vmt файлов (разрабы ксго криво форматируют свои файлы)

antim0118 committed:
Scaling props using UniformScale

antim0118 committed:
Reverted metadatas

Вернул метаданные, чтобы не мозолили глаза в commit changes

lewa-j commented:
Всё круто, но есть одно замечание. Ваш редактор заменил табы на пробелы в отредактированных строках, и всё выравнивание поехало.

antim0118 committed:
Now can load maps in editor (without enabling play mode)

RU: Теперь можно загружать карты на сцену прям в редакторе (без запуска сцены)

antim0118 committed:
Fixed bug with ResourceManager (from last update)

RU: Забыл исправить ResourceManager.

antim0118 committed:
Fixed lightmaps

RU: Пофиксил лайтмапы.

antim0118 committed:
Updated README and gitignore

RU: предупреждение про лайтмапы
и метаданные в игнор (потому что каждый раз 20+ ненужных файлов сюда попадают)

antim0118 commented:
> Всё круто, но есть одно замечание. Ваш редактор заменил табы на пробелы в отредактированных строках, и всё выравнивание поехало.

Да, это Visual Studio, я не могу ничего с этим поделать. Местами вручную поисправлял, чтобы оно в изменения все это не кидал.

+еще обновы докинул.

upd: попробую вручную все поправить

antim0118 committed:
Format.

antim0118 commented:
Вроде исправил. Поначалу я даже не замечал, т.к. VS выравнивал табы и пробелы нормально, Github уже показал такое.

lewa-j commented:
> Да, это Visual Studio, я не могу ничего с этим поделать.

Можешь. Tools->Options->Text Editor->All Languages->Tabs

lewa-j commented:
Теперь можно мержить.
